src/ServerDaemon.chpl

```diff
@@ -607,6 +607,13 @@ module ServerDaemon {
                     break;
                 }
 
+                /*
+                 * If logCommands is true, log incoming request to the .arkouda/commands.log file
+                 */
+                if logCommands {
+                    appendFile(filePath="%s/commands.log".format(this.arkDirectory), "%jt".format(msg));
+                }
```

Comment:
This will need to be updated now that #2632 has been merged. I think all that needs to be is to rebase/merge with master and change this line to:
```suggestion
                if logCommands {
                    appendFile(filePath="%s/commands.log".doFormat(this.arkDirectory), formatJson(msg));
                }
```

Reply:
I confirmed this is the right reformatting once we merge with master. So I went ahead and merged and committed this change